### PR TITLE
feat: add settle cash component

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Settle Cash component added.

## Related issues and discussion
#759 

## Screenshots, if any
Screenshot from 2020-06-16 21-16-41 (1)

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@karantakalkar Please review

File: src/app/organization/tellers/settle-cash/settle-cash.component.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="48%">
+            <mat-label>Currency</mat-label>
+            <mat-select required formControlName="currencyCode">
+              <mat-option *ngFor="let currency of cashierData.currencyOptions" [value]="currency.code">
+                {{ currency.name }}
+              </mat-option>
+              <mat-error *ngIf="settleCashForm.controls.currencyCode.hasError('required')">
```
Comment (karantakalkar) [inline src/app/organization/tellers/settle-cash/settle-cash.component.html:47]: mat error should be outside mat select

File: src/app/organization/tellers/settle-cash/settle-cash.component.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+export class SettleCashComponent implements OnInit {
+
+  /** Cashier data. */
+  cashierData: any;
+  /** Cashier Form. */
+  settleCashForm: FormGroup;
+  /** Minimum Date allowed. */
+  minDate = new Date(2000, 0, 1);
+  /** Maximum Date allowed. */
+  maxDate = new Date();
```
Comment (karantakalkar) [inline src/app/organization/tellers/settle-cash/settle-cash.component.ts:21]: dates should be first 2 variables

**Comment by BLasan** [comment]
@karantakalkar Done

**Comment by karantakalkar** [comment]
@BLasan I 'll merge all your teller-cashier management PRs today, just remove MCs, I'll refactor for changes if required.

**Comment by BLasan** [comment]
> @BLasan I 'll merge all your teller-cashier management PRs today, just remove MCs, I'll refactor for changes if required.

I'll resolve

**Comment by karantakalkar** [comment]
Do it one by one as they may arise again after 1 PR gets merged

**Comment by BLasan** [comment]
@karantakalkar Done

**Pull request merged**